src/platform/utilities/tests/react-hooks.unit.spec.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { expect } from 'chai';
 
-import { usePrevious } from './react-hooks';
```

Comment:
I moved this unrelated test file into the `tests` dir, where it should have been to begin with.